M2/Macaulay2/m2/exports.m2

```diff
@@ -793,6 +794,7 @@ export {
 	"isInputFile",
 	"isIsomorphism",
 	"isListener",
+    "isMember",
```

Comment:
It's going to be pretty confusing to have both `isMember` and `member` defined.

Reply:
Dan, you've mentioned in the past that we should rename `mutable -> isMutable` and similar functions, I think `member -> isMember` would be reasonable as well.

Reply:
Yes, that would be quite reasonable.

Reply:
But changing "member" to "isMember" should be done separately, not in this pull request.